internal/ocm/ocm.go

```diff
@@ -383,3 +391,25 @@ func (csc *clusterServiceClient) ListBreakGlassCredentials(clusterInternalID Int
 	}
 	return BreakGlassCredentialListIterator{request: breakGlassCredentialsListRequest}
 }
+
+func (csc *clusterServiceClient) GetVersion(ctx context.Context, versionName string) (*cmv1.Version, error) {
+	client := cmv1.NewVersionClient(csc.conn, versionName)
```

Comment:
We should be using `arohcpv1alpha1` instead of `cmv1`.  Correct me if you find I'm wrong, but I believe version resources have been ported over to the `aro_hcp/v1alpha1` API.
```suggestion
func (csc *clusterServiceClient) GetVersion(ctx context.Context, versionName string) (*arohcpv1alpha1.Version, error) {
	client := arohcpv1alpha1.NewVersionClient(csc.conn, versionName)
```